### DISCUSSION TITLE
Support Org Mode as editing interface

### DISCUSSION THREAD

**Discussion by mankoff** [magit/forge#580] [Show and tell]
The following snippet changes behavior so that Org Mode is the default major mode when writing new comments on issues. `C-c C-c` is assigned to Org, and you can submit the comment with `C-u C-c C-c`.

```elisp
(defun my/forge-submit-from-org nil
  (interactive)
  (if (equal current-prefix-arg nil)
      (org-ctrl-c-ctrl-c)
    (progn
      (org-export-to-buffer 'gfm (buffer-name))
      (forge-post-submit))))
    
(defun my/forge-create-post-mode nil
  (interactive)
  (use-local-map (copy-keymap org-mode-map))
  (local-set-key (kbd "C-c C-c") 'my/forge-submit-from-org))

(define-derived-mode forge-post-mode org-mode "Forge-Post" "")
(advice-add 'forge-create-post :after #'my/forge-create-post-mode)
```

**Comment by tarsius** [comment]
I also prefer Org, but think Markdown is fine. Of course others have a stronger preference. I don't plan to add support for this to Forge, but its good to have some code out there that users can add to their init file. To that effect I have changed the category of this "discussion" and added links to that category at the top of https://github.com/magit/forge/wiki/Tips-and-Tricks.
